### ISSUE TITLE
HTL Loader brake on the devserver

### ISSUE THREAD

**Issue by mmss1995** [backflip/htl-loader#7]
Hi @backflip,
I use this loader for my ui.frontend project and I wont to setup the module that I can use the .htl file, that I can build and see on the dev server.
I use webpack 5 and htl-loader 4.0.0. This is an extract from my package.json

```javascript
{
  "devDependencies": {
       [...]
         "webpack": "^5.70.0",
         "webpack-cli": "^4.9.2",
         "webpack-dev-server": "^4.7.4",
        "webpack-merge": "^5.8.0"
  },
  "dependencies": {
    "@adobe/htlengine": "^6.4.1",
    "htl-loader": "^4.0.0",
    [...]
  },
}
```

webpack.config.js
```javascript

module: {
        rules: [
            ...
            {
                test: /\.htl$/,
                use: [
                {
                    loader:'htl-loader',
                    options: {
                        includeRuntime: false,
                    },
                },
                ],
            }
        ]
}

```

The includeRuntime option I need to se as false because the loader crash if I set to true. 
my test-component.htl
`<h1 data-sly-use.page="/ui.frontend/mocks/htl/data">${page.title}</h1>`
I use this path because with the version 4.0.0 the use dir was disabled and it's fine. 

But when I try to run the devserver (not the build) he said

![image](https://user-images.githubusercontent.com/32591326/157922148-c4e60be5-94f9-428b-9750-6fc1c6065433.png)

The build go well, but I think if I run the code inside the build was bugged.
Is the first time that I use this loader, so I'm little bit confused.
What I mess up?
Thanks

**Comment by backflip** [comment]
Hey @mmss1995,
I just added a dev server demo: https://github.com/backflip/htl-loader/commit/defb2b9c34c6b124ee3e238a957ba3c00978a4d4

Does this help in any way? Otherwise I would need a complete repository to debug.

Best,
Thomas

**Comment by mmss1995** [comment]
Hi @backflip ,
Ok I try to replicate to my project, but without success. I create a new simple project to see if the problem is the repo but no...So i push that simple project [here](https://github.com/mmss1995/htl-templating). The compling is fine, but it's not able to find the model.

Thanks for your support,
Matteo

**Comment by backflip** [comment]
@mmss1995, uh, definitely a path resolution issue in the loader. Could you check whether the fix in https://github.com/backflip/htl-loader/pull/8 works for you? You can install it via `npm install backflip/htl-loader#bugfix/context`

**Comment by mmss1995** [comment]
@backflip works the path resolution now! I push the [changes on my demo project](https://github.com/mmss1995/htl-templating/commit/ef5b3a5ed6a6952da941ba645bfcb8426a5988d1), if you wont to see it works. Thanks for your support!

**Comment by backflip** [comment]
@mmss1995, released as version `4.1.0`: https://github.com/backflip/htl-loader/releases/tag/v4.1.0
